### PR TITLE
Add zsh completion for 'docker import -m --message'

### PR DESCRIPTION
**Pull request by sdurrheimer**
Ref. #15711

ping @albers for bash

Signed-off-by: Steve Durrheimer s.durrheimer@gmail.com

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @vincentbernat :+1: 

(thanks @sdurrheimer !)

File: contrib/completion/zsh/_docker [hunk 1]
```diff
@@ -641,6 +641,7 @@ __docker_subcommand() {
             _arguments \
                 $opts_help \
                 "($help -c --change)*"{-c,--change=-}"[Apply Dockerfile instruction to the created image]:Dockerfile:_files" \
+                "($help -m --message)"{-m,--message=}"[Set commit message for imported image]:message: " \
```
Comment (vincentbernat) [inline contrib/completion/zsh/_docker]: Missing `-`: `--message=-`

Comment (sdurrheimer) [inline contrib/completion/zsh/_docker]: The message can be set as the next argument. (@see example in https://github.com/monitorjbl/docker/blob/936b2c6afe490b562df4ebea854b116c3b40a9f5/docs/reference/commandline/import.md)

Comment (vincentbernat) [inline contrib/completion/zsh/_docker]: It's just for consistency. We should in fact remove the `-` from everywhere as docker accepts arguments without `=`.

Comment (sdurrheimer) [inline contrib/completion/zsh/_docker]: Ok then I will do another PR for that change.

**Comment by vdemeester** [comment]
Added this to 1.9.0 milestone (as the command is in there).

**Comment by tiborvass** [comment]
LGTM

**Comment by thaJeztah** [comment]
LGTM, thanks @sdurrheimer and thanks for reviewing @vincentbernat

**Pull request merged**